### PR TITLE
Add contrib guide fixes #11

### LINKED ISSUE #11: Add a Contributing Guide
Write a Contributing Guide in the same detail and style as www/.github/CONTRIBUTING.md.

Acceptance Criteria
- [ ]  The README.md links to the Contributing Guide the same way that www/README.md does
- [ ]  The guide is located at .github/CONTRIBUTING.md
- [ ]  The guide includes a table of contents #10
- [ ]  The guide includes a link to the Code of Conduct
- [ ]  The guide includes Issue Reporting Guidelines
- [ ]  The guide includes Pull Request Guidelines
- [ ]  The guide shows Development Setup that includes recommended tools for a good dev experience
- [ ]  The guide explains how to test the project locally (using live-server)
- [ ]  The guide explains Deployments (by pushing/pulling to main)

### PR DESCRIPTION
**Pull request by joeryan**
Added contributors guide, fixes #11

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+## Development Setup
+
+If you want to run this website and make changes to it on your computer, some
+initial setup is recommended.
+
+This section starts with the basics like git and even the recommended editor.
+There may be useful information here even if you're already familiar with
+using git, VS Code, and Node. If you want to jump ahead, though, please read how
+to [customize your environment](#customizing-your-environment).
```
Comment (egillespie) [inline .github/CONTRIBUTING.md]: It looks like there's no section named "Customize Your Enironment". Would you mind removing this link and instead suggest they install the Required Software described below?

**Review by egillespie — CHANGES_REQUESTED** [review]
This reads great! I only noticed one small issue: a link that doesn't seem to work. After that, let's ship this! 🤘

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (joeryan) [reply .github/CONTRIBUTING.md]: Thanks, missed that one.

**Review by egillespie — APPROVED** [review]
Love the double call-out of important sections. Nice touch! 👏  :shipit:

**Pull request merged**